cmd/crictl/container.go

```diff
@@ -646,12 +647,11 @@ var checkpointContainerCommand = &cli.Command{
 	},
 	Action: func(c *cli.Context) error {
 		if c.NArg() == 0 {
-			return fmt.Errorf("ID cannot be empty")
+			return errors.New("ID cannot be empty")
 		}
 		if c.String("export") == "" {
-			return fmt.Errorf(
-				"Cannot checkpoint a container without specifying the checkpoint destination. " +
-					"Use --export=/path/to/checkpoint.tar",
+			return errors.New("Cannot checkpoint a container without specifying the checkpoint destination. " +
+				"Use --export=/path/to/checkpoint.tar",
```

Comment:
The original line break here is better and more readable. We should keep it.

Reply:
True, reverted that particular change.